### DISCUSSION TITLE
Banking, interest and more.

### DISCUSSION THREAD

**Discussion by burning-eggs** [jeremyckahn/farmhand#157] [Ideas]
so, as i was saying in the discord, there should be a banking option. not as in taking a loan but putting cash inside a bank, waiting lets say 15 days and you get a `2% or more` interest.

this will probably change speedrun strategies as one of the speedrun challenge is to get the millionaire achievement in the lowest amount of time

one of the features for banking that will probably be interesting is that you cant deposit bank money

**Comment by jeremyckahn** [comment]
I like this idea! Thank you for contributing it. I think investment mechanics fit well with Farmhand's concept! I'd like to keep this feature simple and straightforward so that players don't have to do too much complex math to enjoy it. That said, what do you think about giving the player interest every day (rather than intervals like ~15)?

My concern is that it'll get confusing for players when they put in $100 on on day 1, then $50 on day 3. What happens when day 15 comes? Do they get interest on the initial $100, the second installment of $50, or some prorated amount of the latter? I'd like to avoid these issues entirely, as it could be confusing for players, and certainly more complex for us to implement.

I'd like to get your thoughts on simplifying this idea a bit @flying-dojo!

**Reply by burning-eggs** [reply]
they should get the <full amount * 2%> they have deposited in the bank on the next day

i really dont have anything else to say as im really burned out today

**Reply by jeremyckahn** [reply]
>they should get the <full amount * 2%> they have deposited in the bank on the next day

Yep that works for me!

Sorry to hear you're feeling burnt out. Don't stress yourself with working on Farmhand — this is all just for fun, so take your time with things!

**Reply by lstebner** [reply]
i like this idea, and i think it would be pretty easy to add to the bank UI a "savings account" area. maybe we could add some feature into it like a minimum needed to collect interest? instead of the payout only after X days.

another idea is instead of just a savings account it can be something like an investment, or bond the player buys that gains value after X number days of maturing. maybe you can buy a share of a crop and then the fluctuating market values come into play? that's a really half-baked idea i'm just throwing out here

**Reply by jeremyckahn** [reply]
I'm generally onboard with incorporating more investment mechanics like what you're both describing. These all fit well with Farmhand's general concept. One thing I want to make sure we prioritize is keeping things straightforward, even if that means mechanics become somewhat oversimplified. I think that's a fine line to walk with what we're talking about, because investment mechanics are inherently complex and I barely understand them myself a lot of times.

To put it another way, the general idea of "buy low and sell high" is simple enough where most people can understand it. It sounds like we're hinting at something like buying crop futures (also described by [this Reddit post](https://www.reddit.com/r/incremental_games/comments/nil6c1/exciting_updates_for_farmhand_in_1100/gz478xi/?utm_source=reddit&utm_medium=web2x&context=3)). These more advanced investment concepts can definitely make for fun game mechanics, but they can also get overcomplicated to the point where players need to keep a spreadsheet or pull out a calculator to succeed with them. Generally speaking, I'd like to keep the mechanics in Farmhand simple enough where all of the rules can be thought of as "do X to make more money, avoid Y to prevent losing money" (like "buy low and sell high").

Hopefully makes sense! 😅
